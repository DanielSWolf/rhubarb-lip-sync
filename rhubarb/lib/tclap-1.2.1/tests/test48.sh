#!/bin/sh

# success
../examples/test8  -s=aaa homer marge bart -- one two > tmp.out 2>&1

if cmp -s tmp.out $srcdir/test48.out; then
	exit 0
else
	exit 1
fi

