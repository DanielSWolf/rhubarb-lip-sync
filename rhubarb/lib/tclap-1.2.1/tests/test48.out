[  ] 0  bart
[  ] 1  one
[  ] 2  two
for string we got : aaa
for ulabeled one we got : homer
for ulabeled two we got : marge
for bool B we got : 0
